### PR TITLE
Modify schema of scenario list

### PR DESCRIPTION
**Pull request by rouille**
[Pull Request doc](https://breakthrough-energy.github.io/docs/user/git_guide.html#d-pull-request)

### Purpose
Update the schema of the scenario list (csv and sql)

### What the code is doing
Add an extra field `grid_model`

### Testing
How did you test this change (unit/functional testing, manual testing, etc.)? 

### Where to look
Look for an extra field named `grid_model` in the scenario list template.

### Time estimate
2min

### CODE REVIEW

**Review by jon-hagg — APPROVED** [review]

File: powersimdata/data_access/sql/schema.sql [hunk 1]
```diff
@@ -20,5 +20,6 @@ CREATE TABLE IF NOT EXISTS scenario_list(
   interval text not null,
   engine text not null,
   runtime text,
-  infeasibilities text
+  infeasibilities text,
+  grid_model text
```
Comment (jon-hagg) [inline powersimdata/data_access/sql/schema.sql:24]: Might want this to be `not null` unless we have a default, but it's fine for now

Comment (rouille) [reply powersimdata/data_access/sql/schema.sql]: Done

File: powersimdata/utility/templates/ScenarioList.csv [hunk 2]
```diff
@@ -1 +1 @@
-id,plan,name,state,interconnect,base_demand,base_hydro,base_solar,base_wind,change_table,start_date,end_date,interval,engine,runtime,infeasibilities
+id,plan,name,state,interconnect,base_demand,base_hydro,base_solar,base_wind,change_table,start_date,end_date,interval,engine,runtime,infeasibilities,grid_model
```
Comment (danielolsen) [inline powersimdata/utility/templates/ScenarioList.csv:1]: Should the new `grid_model` column live next to the `interconnect` column? I think we factored out all the hard-coded column numbers, so changing the position of the other columns shouldn't matter.

Comment (rouille) [reply powersimdata/utility/templates/ScenarioList.csv:1]: As you wish. I will have to switch the order in the Create class.

Comment (rouille) [reply powersimdata/utility/templates/ScenarioList.csv]: Done

**Review by danielolsen — APPROVED** [review]

**Pull request merged**